docs/cloud/ui/interactive-api.md

```diff
@@ -1,6 +1,116 @@
 # Interactive API
 
 
-The Interactive API is an embedded GraphQL client that allows you to access all of your data with complete flexibility. Thanks to features like schema introspection and automatic authentication, the interactive API makes it simple to access Cloud. You can also see the GraphQL API documentation directly inline.
+The Interactive API is an embedded GraphQL client that allows you to access all of your data with complete flexibility. Thanks to features like schema introspection and automatic authentication, the interactive API makes it simple to access Cloud. You can also see the GraphQL API documentation directly inline.  
+
+![](/cloud/ui/interactive-api.png)
+
+
+### Basic Query
+
+Let's start with an example of how you might find basic information about your flows.
+
+```graphql
+query {
+    flow {
+        name
+        id
+    }
+}
+```
+Notice that when we want to get the `flow` object, we must wrap it in a `query` block. This tells GraphQL that we're _retrieving_ something from the schema, as opposed to a `mutation` block which allows us to manipulate our data.
+
+We've told the server to return a list of `flow` objects, each containing their respective `name` and `id` fields. As you interact with the schema in the editor window, you'll see an autocomplete window that will show hints of other available fields, which you can select to insert into your query. 
+
+![](/cloud/ui/dropdown-on-interactive-api.png)
+
+You can see more information about GraphQL mutations and queries in the [GraphQL Docs](https://graphql.org/learn/)
+
+### Nested Query
+
+Let's build on our earlier query to get information about the tasks associated with our flows. One way to accomplish this would be to use a nested query:
+
+```graphql
+query {
+  flow {
+    name 
+    id
+    tasks {
+      name
+    }
+  }
+}
+```
+We've nested `tasks` within the `flow` object, which tells the server to retrieve tasks only within the context of each flow that it returns.
```

Comment:
```suggestion
```

We've nested `tasks` within the `flow` object, which tells the server to retrieve tasks only within the context of each flow that it returns.
```

Reply:
This actually does not affect rendering